static/docs/commands-reference/pipeline/pipeline.md

```diff
@@ -0,0 +1,28 @@
+# pipeline
+
+A set of commands to manage pipelines: [show](#show) - visualize or
+[list](#list) - list existing pipelines.
```

Comment:
this is not the most recent version of the file, please double check that conflict is resolved properly

Reply:
This version of the file is now in the master branch, we just moved this file to the pipeline subfolder.